sos/report/plugins/fibrechannel.py

```diff
@@ -33,4 +33,17 @@ def setup(self):
         if self.get_option('debug'):
             self.add_copy_spec(self.debug_paths)
 
+        self.add_cmd_output([
+            "hbacmd listhbas",
+            "hbacmd ServerAttributes"
+        ])
+
+        # collect Hbaattributes and Portattributes of WWN
+        listhbas = self.collect_cmd_output("hbacmd listhbas")
+        if listhbas['status'] == 0:
+            for line in listhbas['output'].splitlines():
+                if 'Port WWN       :' in line:
```

Comment:
Can this just be `if 'Port WWN' in line:`? I'm hesitant to include a static amount of whitespacing here, as that is far more prone to breaking on minor output changes.

Reply:
ok sure..will make those changes and send new version of patch..
Thank you